### PR TITLE
Add RTCConfiguration compat info

### PR DESCRIPTION
**Pull request by bunnybooboo**
Adding RTCConfiguration compat data. First time adding notes!

### CODE REVIEW

File: api/RTCCertificate.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
```
Comment (teoli2003) [inline api/RTCCertificate.json]: This file shouldn't be there.

You need either to rebase your branch to master (your local master), or create a new branch out of your local master.

File: api/RTCConfiguration.json [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "bundlePolicy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "certificates": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "iceCandidatePoolSize": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 5]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "iceServers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 6]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "iceTransportPolicy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 7]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "peerIdentity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 8]
```diff
@@ -0,0 +1,394 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "rtcpMuxPolicy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCConfiguration",
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: This URL is wrong

File: api/RTCConfiguration.json [hunk 9]
```diff
@@ -0,0 +1,394 @@
...
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "57",
+              "notes": [
+                "Default for rtcpMuxPolicy is require"
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: Default for <code>rtcpMuxPolicy</code> is <code>require</code>

Please add \<code> around the two tokens

File: api/RTCConfiguration.json [hunk 10]
```diff
@@ -0,0 +1,394 @@
...
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "44",
+              "notes": [
+                "Default for rtcpMuxPolicy is require"
```
Comment (teoli2003) [inline api/RTCConfiguration.json]: Add \<code> around the two tokens

**Comment by bunnybooboo** [comment]
New branch in attempt to solve reabse issue https://github.com/mdn/browser-compat-data/pull/958

**Pull request closed without merging**